cgi-bin/sitemap.js

```diff
@@ -12,9 +12,9 @@
 const algoliasearch = require('algoliasearch');
 const { logger } = require('@adobe/openwhisk-action-logger');
 const { wrap } = require('@adobe/openwhisk-action-utils');
-
 const Downloader = require('@adobe/helix-pipeline/src/utils/Downloader.js');
 const { MountConfig, IndexConfig } = require('@adobe/helix-shared');
+const { getOriginalHost } = require('../src/utils');
```

Comment:
I sometimes see `require('../src/file.js')` and somtimes `require('../src/file')`, which are both working: do we have a convention on this?

Reply:
I'm not sure, I usually omit the extension. Must have seen it elsewhere 😄